test/common/test/api_utils_test.py

```diff
@@ -0,0 +1,203 @@
+"""
+Copyright 2016, EMC, Inc.
+
+Author(s):
+"""
+import sys
+import subprocess
+
+# set path to common libraries
+sys.path.append(subprocess.check_output("git rev-parse --show-toplevel", shell=True).rstrip("\n") + "/test")
+
+import unittest
+
+#CIT
+from modules.logger import Log
+from on_http_api2_0 import ApiApi as Api
+from config.api1_1_config import config as config_old   #TODO remove when 2.0 worklfow API is implemented
+from config.api2_0_config import config
+from json import loads
+
+from common.api_utils import api_node_select_from_config, api_node_select, api_validate_node_pollers
+
+LOG = Log(__name__)
+
+from nose.plugins.attrib import attr
+@attr(all=True, regression=True, smoke=True)
+
+class NodesTests(unittest.TestCase):
+
+    def setUp(self):
+        self.__client_old = config_old.api_client
+        self.__client = config.api_client
+    def __get_data(self):
+        return loads(self.__client.last_response.data)
+
+    def test_all_list(self):
+        Api().nodes_get_all()
+        node_list = self.__get_data()
+
+        id_list = api_node_select(self.__client)
+        self.assertNotEqual(len(node_list), len(id_list), msg='Id list not equal to node list')
```

Comment:
please add the values of the failed comparison into the assert (like you do in test_type_list()